regulations/static/regulations/js/source/views/comment/comment-confirm-view.js

```diff
@@ -59,8 +59,11 @@ var CommentConfirmView = Backbone.View.extend({
     this.replaceTemplate('.save-pdf', {url: url});
   },
 
+// Changes text and color of background when tracking number is received, hides wait message
   setTrackingNumber: function(number) {
     this.replaceTemplate('.tracking-number .status', {number: number});
+    $('.status').addClass('tracking-number-retrieved');
+    $('.status-waiting').hide();
```

Comment:
Let's avoid accessing the global DOM here and scope our queries instead -- does
```javascript
this.$el.find('.status').//etc
```
work?